### PR TITLE
Use eager-fetch to get brightcove data

### PR DESCRIPTION
**Pull request by ironsidevsquincy**
Retry 3 times

Also,  add duration to video attachment, and story package

@i-like-robots

### CODE REVIEW

File: server/models/video.js [hunk 1]
```diff
@@ -72,7 +70,8 @@ function applyModel (brightcove) {
 				mediaType: 'video/mp4',
 				url: rendition.url,
 				width: rendition.frameWidth,
-				height: rendition.frameHeight
+				height: rendition.frameHeight,
+				duration: rendition.videoDuration
```
Comment (ironsidevsquincy) [inline server/models/video.js]: Need the video duration so adding this in too, does that gel with the data model @i-like-robots?

Comment (i-like-robots) [inline server/models/video.js]: Nope, but I don't think that matters - we don't use it for search or anything where mapping optimizations are required.

File: server/models/video.js [hunk 2]
```diff
@@ -52,18 +53,15 @@ function applyModel (brightcove) {
 	);
 
 	return {
-		// TODO: this should come from CAPI soon
-		title: brightcove.name,
+		// TODO: these should come from CAPIv2
 		alternativeTitles: {},
-		// TODO: this should come from CAPI soon
 		bodyXML: `<p>${brightcove.longDescription}</p>`,
 		bodyHTML: `<p>${brightcove.longDescription}</p>`,
```
Comment (i-like-robots) [inline server/models/video.js]: Could we take the `bodyXML` field from CAPI?

Comment (ironsidevsquincy) [inline server/models/video.js]: I did that, but then noticed that its format is different to the format of 'normal' articles, e.g. it doesn't have a wrapping `<p>`

File: server/models/video.js (hunk 1 above)
Comment (ironsidevsquincy) [inline server/models/video.js]: ahh, but I'd like to use it for search, e.g. get all video content that is less than a minute long. Maybe it should live at the top level, `videoDuration` or something?

File: server/models/video.js (hunk 2 above)
Comment (i-like-robots) [inline server/models/video.js]: Could regex? Whatever you think is better anyway

**Pull request merged**